jenkins.yaml

```diff
@@ -0,0 +1,3 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code plugin\n\n"
+  numExecutors: 1
```

Comment:
why do you have this in jenkins.yaml and executors.groovy?

Reply:
Good catch, I will rewrite the history: `executors.groovy` is not used.